outport/host/driver.go

```diff
@@ -102,6 +114,21 @@ func (o *hostDriver) handleAction(args interface{}, topic string) error {
 	return nil
 }
 
+// RegisterHandlerForSettingsRequest will register the handler function for the settings request
+func (o *hostDriver) RegisterHandlerForSettingsRequest(handlerFunction func()) error {
+	return o.payloadProc.SetHandlerFunc(handlerFunction)
+}
+
+// CurrentSettings will send the current settings
+func (o *hostDriver) CurrentSettings(config outport.OutportConfig) error {
```

Comment:
rename to `SetCurrentSettings`

Reply:
Yes, we can rename to SetCurrentSettings

Reply:
done